opencti-platform/opencti-front/src/private/components/settings/users/UserConfidenceLevel.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { User_user$data } from '@components/settings/users/__generated__/User_user.graphql';
+import { Link } from 'react-router-dom-v5-compat';
+import { ReportGmailerrorred } from '@mui/icons-material';
+import Tooltip from '@mui/material/Tooltip';
+import { InformationOutline } from 'mdi-material-ui';
+import Box from '@mui/material/Box';
+import { useFormatter } from '../../../../components/i18n';
+
+type Data_UserConfidenceLevel = User_user$data['user_confidence_level'];
+type Data_EffectiveConfidenceLevel = User_user$data['effective_confidence_level'];
+
+type UserConfidenceLevelProps = {
+  confidenceLevel?: Data_UserConfidenceLevel | Data_EffectiveConfidenceLevel
+};
+
+const UserConfidenceLevel: React.FC<UserConfidenceLevelProps> = ({ confidenceLevel }) => {
+  const { t_i18n } = useFormatter();
+
+  if (!confidenceLevel) {
+    return (
+      <Tooltip
+        title={t_i18n("No confidence level found in this user's groups, and no confidence level defined at the user level.")}
+      >
+        <ReportGmailerrorred fontSize={'small'} color={'error'}/>
+      </Tooltip>
+    );
+  }
+
+  // TODO: add overrides in a tooltip when in use
```

Comment:
TODO later ?

Reply:
yes normal, it will come in 6.1 with #5694